controllers/templatesControllers.js

```diff
@@ -39,7 +56,7 @@ function editTemplateHandler(req, res) {
     req
         .on(`data`, data => editedTemplate = JSON.parse(data.toString()))
```

Comment:
your mixing layers and technologies here
```
req.on('data', ...).on('end' ...)..
```
is a node HTTP approach, while you are using Express here, please read how to get the body in  Express.

Reply:
sorry my bad, this comment is not relevant to you as these changes do not occur by this commit.